app/views/state_file/questions/nj_college_dependents_exemption/edit.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+    title = t(".title")
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t(".label_1") %></p>
```

Comment:
[dust] Calling this "label" is a little confusing to read. I would expect a label to be something that is used in a label element. Maybe subheader? Description?